src/Modules/ConfigOptions.lua

```diff
@@ -726,12 +727,20 @@ return {
 	end },
 	{ var = "conditionIgnited", type = "check", label = "Are you Ignited?", ifCond = "Ignited", implyCond = "Burning", tooltip = "This also implies that you are Burning.", apply = function(val, modList, enemyModList)
 		modList:NewMod("Condition:Ignited", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
+		modList:NewMod("Condition:Burning", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
 	end },
 	{ var = "conditionChilled", type = "check", label = "Are you Chilled?", ifCond = "Chilled", apply = function(val, modList, enemyModList)
 		modList:NewMod("Condition:Chilled", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
 	end },
+	{ var = "conditionChilledEffect", type = "count", label = "Effect of Chill:", ifOption = "conditionChilled", apply = function(val, modList, enemyModList)
+		modList:NewMod("ChillVal", "OVERRIDE", val, "Chill", { type = "Condition", var = "Chilled" })
+	end },
+	{ var = "conditionSelfChill", type = "check", label = "Did you chill yourself?", ifOption = "conditionChilled", apply = function(val, modList, enemyModList)
+		modList:NewMod("Condition:ChilledSelf", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
```

Comment:
If we're providing an override anyway, do we really need this extra checkbox?

Reply:
The override only says how much you're chilled for. When you chill yourself, effects that are normally applied on the enemy (increased effect of chill) now apply to you. We need to specify whether you caused the chill or if an enemy/effect did.